Configure Elasticsearch - clarify elasticsearch.yml configuration for self-managed clusters (#7847)
elastic/docs-content: deploy-manage/stack-settings.md

#### Elasticsearch (`elasticsearch.yml`)

Most settings can be changed on a running cluster using the [Cluster update settings]({{es-apis}}operation/operation-cluster-put-settings) API, while others must be configured statically in `elasticsearch.yml` and require a cluster restart. To learn more, refer to [Dynamic and static {{es}} settings](#static-dynamic).

> In a multi-node cluster, `elasticsearch.yml` settings must be applied to each relevant node, which in most cases means all nodes.

Author: ```suggestion
In a multi-node cluster, `elasticsearch.yml` settings must be applied across all relevant nodes.
```
